### ISSUE TITLE
Installed GCC by setup-gcc not set to default GCC

### ISSUE THREAD

**Issue by CGarces** [egor-tensin/setup-gcc#3]
I'm trying to implement a build matrix for gcc [9,10,11]

The striped version of my workflow is something like this

```yaml
  build:
    needs: commontasks
    runs-on: ubuntu-latest
    strategy:
      fail-fast: false
      matrix: 
        gcc_version: [9,10,11]
    steps:
    - name: Set up GCC
      uses: egor-tensin/setup-gcc@v1
      with:
        version: ${{matrix.gcc_version }}
    - uses: actions/checkout@v2
    - name: build
      run: gcc --version

```
My expectation is receive a different value for **gcc --version** on each build stage but I always get "gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0" that is the one installed by default on 20.04

The full workflow is at 
https://github.com/Mange/rtl8192eu-linux-driver/blob/test_gcc/.github/workflows/build.yml

For example this logs are for GCC 11 and 10 but both shows "9.3.0"
https://github.com/Mange/rtl8192eu-linux-driver/runs/3517256317?check_suite_focus=true
https://github.com/Mange/rtl8192eu-linux-driver/runs/3517256332?check_suite_focus=true

**Comment by egor-tensin** [comment]
I'll look into making a symlink at /usr/local/bin/gcc. Meanwhile, as a workaround, please use `cc --version`. `cc` is used by default by GNU Make & CMake.

**Comment by egor-tensin** [comment]
There's an undocumented output `gcc` to this action also, it will contain the executable name with the version, so you can also use `${{ steps.setup-gcc.outputs.gcc }} --version`.

**Comment by CGarces** [comment]
> There's an undocumented output `gcc` to this action also, it will contain the executable name with the version, so you can also use `${{ steps.setup-gcc.outputs.gcc }} --version`.

Thanks, solved using cc.
**make KVER=$KVER CC=cc**

Maybe something like "sudo update-alternatives" should work fine instead for try to made the symlinks by hand

**Comment by egor-tensin** [comment]
Fixed in version v1.2 (v1 now points to it). Please feel free to reopen if there're any problems.
